Neos.Http.Factories/composer.json

```diff
@@ -7,9 +7,9 @@
         "MIT"
     ],
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^8.0",
```

Comment:
This would change the required PHP version in a patch level release!?

Reply:
But that change is in the `neos/http-factories` package, so Flow 7.3.x could still be installed with PHP < 8. With this. Or am I missing something here?

Reply:
IDK. I guess so. But I was just checking the history of this package composer.json and was assuming this is right, as it was released like that before.

Reply:
As of https://github.com/neos/http-factories/commit/a15a3853780bbb37617f0080e05ea954f00f983b, to be precise.

But… that cannot be correct, as https://docs.neos.io/guide/installation-development-setup/system-requirements#php-version-matrix as well as https://github.com/neos/flow-development-collection/blob/1d64e6a72c2e419e7dd9a5546bf4ac0735a70001/composer.json say it's compatible with PHP 7.3 and up.

So it seems that back then this was a mistake.

Reply:
I see! But `guzzlehttp/psr7@^2.1.1` is fine?

Reply:
So actually this commit must not have to get into 7.3:
https://github.com/neos/http-factories/commit/a15a3853780bbb37617f0080e05ea954f00f983b

Reply:
> I see! But `guzzlehttp/psr7@^2.1.1` is fine?

Huh… not sure, would need to look it up.

> So actually this commit must not have been gone into 7.3: [neos/http-factories@a15a385](https://github.com/neos/http-factories/commit/a15a3853780bbb37617f0080e05ea954f00f983b)

I'd say so, yes!

Reply:
@kdambekalns I changed the dependencies accordingly. Could you please approve it, if this is fine for you?